client/state/support/reducer.js

```diff
@@ -1,11 +1,39 @@
 /** @format */
 
 /**
- * External dependencies
+ * Internal dependencies
  */
+import { createReducer, combineReducers } from 'state/utils';
+import {
+	SUPPORT_SESSION_ACTIVATE,
+	SUPPORT_SESSION_EXPIRE,
+	SUPPORT_USER_ACTIVATE,
+	SERIALIZE,
+	DESERIALIZE,
+} from 'state/action-types';
 
-import { combineReducers } from 'state/utils';
-import { SUPPORT_USER_ACTIVATE } from 'state/action-types';
+export const SUPPORT_SESSION_STATE = {
+	NONE: 'none',
+	ACTIVE: 'active',
+	EXPIRED: 'expired',
+};
```

Comment:
Thanks for letting me know. What I really want is to express an enumeration, but since AFAIK, we cannot and aren't using any sort of type checker, we can just change each value to a standalone constant.

Reply:
use your judgement. we've mostly transitioned away from this pattern of setting properties on an object just because `webpack` and `uglify` can give us some free type checking and optimization if we break them into named exports